picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java

```diff
@@ -30,6 +34,9 @@
     implements LoaderManager.LoaderCallbacks<Cursor> {
   private SampleContactsAdapter adapter;
 
+  private static final String sReadContacts = Manifest.permission.READ_CONTACTS;
+  private static final int sReadContactsCode = 100;
```

Comment:
AOSP's style is a historical accident and shouldn't be used anywhere except inside AOSP.